### PR TITLE
Document Quotas

### PR DESCRIPTION
**Pull request by dadgar**

### CODE REVIEW

File: website/source/guides/quotas.html.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+Name        = default-quota
+Description = Limit the shared default namespace
+Limits      = 1
+
+Quota Limits
+Region  CPU Usage   Memory Usage
+global  500 / 2500  256 / 1000
+```
+
+We can see the newly created job is accounted against the quota specification
```
Comment (chelseakomlo) [inline website/source/guides/quotas.html.md]: "accounted against"- maybe either "accounted for" or "counted against"?

File: website/source/guides/quotas.html.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+namespaces can apply the `prod-api` quota specification.
+
+When a quota specification is attached to a namespace, all resource usage by
+jobs in the namespaces are accounted toward the quota limits. If the resource is
+exhausted, allocations with the namespaces will be queued until resources become
+available by either other jobs finishing or the quota being expanded.
+
+## Working with Quotas
+
+For specific details about working with quotas, see the [quotas
```
Comment (chelseakomlo) [inline website/source/guides/quotas.html.md]: Maybe add something about how ACLs are useful to make sure quotas are not "self serve"? i.e, if ACLs are enabled, then only administrators with either a management token or a token with the correct permissions can create a quota.

File: website/source/api/quotas.html.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+[blocking queries](/api/index.html#blocking-queries) and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | ACL Required         |
+| ---------------- | -------------------- |
+| `YES`            | `quota:read`<br>`namespace:*` if namespace has quota attached|
+
+### Parameters
+
+- `:quota` `(string: <required>)`- Specifies the quota specification to query.
```
Comment (chelseakomlo) [inline website/source/api/quotas.html.md]: Maybe add that this identifier is the quote name.

File: website/source/api/quotas.html.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+[blocking queries](/api/index.html#blocking-queries) and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | ACL Required |
+| ---------------- | ------------ |
+| `NO`             | `quota:write` |
+
+### Parameters
+
+- `:quota` `(string: <required>)`- Specifies the quota specification to delete.
```
Comment (chelseakomlo) [inline website/source/api/quotas.html.md]: Where the identifier is the quota name (if this is the identifier).

File: website/source/guides/quotas.html.md (hunk 2 above)
Comment (dadgar) [reply website/source/guides/quotas.html.md]: Good idea, added to the acl section.

**Pull request merged**